src/connectors/ona-api/aggregateFormData.js

```diff
@@ -245,6 +295,6 @@ export default function aggregateFormData(layerData, locations, filterOptions) {
   data = processFilters(layer, filterOptions);
 
   // Process data
-  aggregatedData = processFormData(data, layer.property, layer.aggregate);
+  aggregatedData = processFormData(data, layer.property, layer.aggregate, layer.extraProps);
```

Comment:
Actually, if the `extraProps` array only pertains to the data aggregation, can we nest it into `layer.aggregate` in the layer spec, keep `processFormData` to 3 parameters, and define `extraProps` once in the function?